nav2_lifecycle_manager/test/launch_lifecycle_test.py

```diff
@@ -32,7 +32,7 @@ def generate_launch_description():
             output='screen',
             parameters=[{'use_sim_time': False},
                         {'autostart': False},
-                        {'bond_timeout_ms': 0},
+                        {'bond_timeout': 0},
```

Comment:
I think this needs to be `0.0` now (might be why the CI test isn't returning a result for the lifecycle tests)